components/post_view/message_attachments/message_attachment/message_attachment.jsx

```diff
@@ -365,6 +365,40 @@ export default class MessageAttachment extends React.PureComponent {
             );
         }
 
+        let footer;
+        if (attachment.footer) {
+            let footerIcon;
+            if (attachment.footer_icon) {
+                const footerIconMetadata = this.props.imagesMetadata[attachment.footer_icon];
+
+                footerIcon = (
+                    <ExternalImage
+                        src={attachment.footer_icon}
+                        imageMetadata={footerIconMetadata}
+                    >
+                        {(footerIconUrl) => (
+                            <img
+                                alt={'attachment footer icon'}
+                                className='attachment__footer-icon'
+                                src={footerIconUrl}
+                                height='14'
+                                width='14'
```

Comment:
Slack does renders it at 16px by 16px. Thoughts on syncing this behaviour to slacks?

Reply:
@hanzei I originally had these at 16x16, but reduced them to 14x14 to match the [author icon](https://github.com/mattermost/mattermost-webapp/blob/97cba9d5671cd11f78da4c9ba903f892397871f3/components/post_view/message_attachments/message_attachment/message_attachment.jsx#L267).  I actually like the look of the footer icon at 16x16 a bit better - I'll go with this unless I hear otherwise from you 👍